.github/workflows/unit-tests.yml

```diff
@@ -97,48 +97,3 @@ jobs:
           git commit -am "Look at this (photo)Graph *in the voice of Nickelback*" || true
           git push
           git push origin ${{ github.head_ref }}
-
-  npm-test-unit-components:
```

Comment:
All jest unit test component stuff is merged into `vitest` since it is compatible and supports more of our repository level configurations.